.circleci/config.yml

```diff
@@ -5,11 +5,19 @@ jobs:
     docker:
       - image: circleci/rust:1.41-stretch
     steps:
-      - run: rustup toolchain install nightly
+      - checkout
+      - run:
+          name: Setup Rust toolchain from ./rust-toolchain
+          command: echo "RUSTUP_TOOLCHAIN=$RUSTUP_TOOLCHAIN" && rustup show
```

Comment:
👆 installs missing toolchains and activates the desired toolchain from [those sources](https://github.com/rust-lang/rustup#override-precedence)